packaging/nuget/nservicebus.ravendb.nuspec

```diff
@@ -16,7 +16,7 @@
     <tags>nservicebus servicebus msmq cqrs publish subscribe ravendb</tags>
     <dependencies>
       <dependency id="NServiceBus" version="[5.2.8, 6.0.0)" />
-      <dependency id="RavenDB.Client" version="[3.0.3800, 4.0.0)" />
+      <dependency id="RavenDB.Client" version="[3.0.3800, 3.1.0)" />
```

Comment:
We've now released NServiceBus.RavenDB 3.2.0 to support RavenDB 3.5 with NServiceBus 5.